spec/std/big/big_decimal_spec.cr

```diff
@@ -148,6 +148,7 @@ describe BigDecimal do
       BigDecimal.new(-1) / BigDecimal.new(0)
     end
     BigDecimal.new(1).should eq(BigDecimal.new(1) / BigDecimal.new(1))
+    BigDecimal.new(10).should eq(BigDecimal.new(100, 1) / BigDecimal.new(100000000, 8))
```

Comment:
Not for this PR, but all of these assertions are written backwards: what one expects to get should be at the right-hand side.